### DISCUSSION TITLE
Search for docs / tutorials

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [vanjs-org/van#320] [Ideas]
Search for docs / tutorials on vanJS.org

**Comment by Tao-VanJS** [comment]
The VanJS tutorial is literally a single web page. I think using `Find` via your browser will just serve what you need.

**Comment by HEAVYPOLY** [comment]
That works for tutorials page but not for searching through the entire site, for example I was looking for 'router' and found it in Home section.

**Comment by Tao-VanJS** [comment]
Maybe try Google "`<keywork> site:vanjs.org`"?

**Comment by HEAVYPOLY** [comment]
Thanks, that works for now
